CONTRIBUTING.md

```diff
@@ -42,70 +42,22 @@ Merging Pull Requests (for project contributors with write access)
 
 ### Pre-requisites
 
-This project contains Git submodules which are required to build. After cloning this repo, run `git submodule update
---init --recursive`.
-
-### CMake
-CMake version 3.12 or higher is required to build these libraries. Download and install CMake from the project's
+#### CMake
+CMake version 3.15 or higher is required to build these libraries. Download and install CMake from the project's
 [website](https://cmake.org/download/).
 
-### Vcpkg
-Vcpkg is required to download project dependencies. To get started, first clone vcpkg to a location on your system and
-run the bootstrapping script.
+#### Vcpkg
+Vcpkg can be used to have the Azure for SDK for CPP dependencies installed into a specific folder in the system instead of globally installing them.
```

Comment:
```suggestion
Vcpkg can be used to install the Azure SDK for CPP dependencies into a specific folder on the system instead of globally installing them.
```